docs/tutorial.md

```diff
@@ -595,6 +598,33 @@ You must reconfigure any `image` resource definitions in your `PipelineResources
   example configuration at <https://github.com/mgreau/tekton-pipelines-elastic-tutorials>.
 - To learn more about obtaining logs, see [Logs](logs.md).
 
+### Prerequisites: Minikube
+
+Complete these prerequisites to run this tutorial locally using Minikube:
+
+- Install the [required tools](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md#requirements).
+- Install [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) and start a sesion as follows:
+```bash
+minikube start --memory 10240 --cpus 6 
```

Comment:
fwiw I develop Tekton in a minikube with 6GB RAM and 2 cpus and things run really nicely. (this is using the virtualbox driver on mac)

Reply:
Happy to lower this -- I just mirrored the specs that were listed from the Docker for Desktop section : "Install Docker for Desktop and configure it to use six CPUs, 10 GB of RAM and 2GB of swap space."